bitbucket-pipelines.yml

```diff
@@ -5,16 +5,44 @@ definitions:
     docker:
       memory: 3072
   steps:
-    - step: &test
-        name: Test
+    - step: &install
+        name: ⚙️ Install
         script:
-          - FORCE_COLOR=true
           - npm ci
+        artifacts:
+          - node_modules/**
+    - step: &lint
+        name: 🧹 Lint
+        script:
           - npm run lint
-          - npm run check-types
+    - step: &test
+        name: 🧪 Test
+        script:
           - npm run test
+    - step: &deploy-temp-stack
+        name: 🏗️ Test Deployment
+        script:
+          - TMP_STAGE=$(cat /dev/urandom | tr -dc 'a-z' | head -c3 | grep -vE 'prd|stg')
```

Comment:
Won't a lot of services require SSM values with the stage as part of the path? So the deploy will fail since those don't exist?

Reply:
I spoke to Todd about this (and from what I've seen recently) we seem to be moving away from putting the stage name in the ssm parameter name. As they're in different AWS accounts anyway